sentry/sentry.conf.example.py

```diff
@@ -265,13 +268,18 @@ def get_internal_network():
 ########
 
 SENTRY_OPTIONS["mail.list-namespace"] = env("SENTRY_MAIL_HOST", "localhost")
-SENTRY_OPTIONS["mail.from"] = f"sentry@{SENTRY_OPTIONS['mail.list-namespace']}"
+SENTRY_OPTIONS["mail.from"] = env('SENTRY_MAIL_FROM', f"sentry@{SENTRY_OPTIONS['mail.list-namespace']}")
+SENTRY_OPTIONS["mail.port"] = env('SENTRY_MAIL_PORT', 465)
+SENTRY_OPTIONS["mail.username"] = env('SENTRY_MAIL_USERNAME', '')
+SENTRY_OPTIONS["mail.password"] = env('SENTRY_MAIL_PASSWORD', '')
+SENTRY_OPTIONS["mail.use-tls"] = env('SENTRY_MAIL_USE_TLS', 'false').lower() in ('true')
+SENTRY_OPTIONS["mail.use-ssl"] = env('SENTRY_MAIL_USE_SSL', 'false').lower() in ('true')
```

Comment:
Shouldn't these defined on the `config.yml` file, instead of here? Or do you disable those on `config.yml` and define the mail configurations here instead?